travis_scripts/trigger-dockerhub.sh

```diff
@@ -25,7 +25,7 @@ if [[ -z "$DOCKERHUB_TOKEN" ]] ; then
     exit 1
 fi
 
-if [[ -n "$COVERAGE" ]] ; then
+if [[ -n "$COVERAGE" && $COVERAGE -ne 0 ]] ; then
```

Comment:
This condition matches the one [here](https://github.com/travis-perl/helpers/blob/557e0bf2e69f2514a52d678326757bbf29106903/init#L88):

```shell
 if [ -n "$COVERAGE" ] && [ "$COVERAGE" != "0" ]; then
```

so this should be correct.